tests/integrations/subprocess/test_Subprocess__Popen.py

```diff
@@ -5,9 +5,11 @@
 from briefcase.console import Log
 
 
-def test_call(mock_sub, capsys):
+@pytest.mark.parametrize("platform", ["linux", "darwin", "win32"])
+def test_call(mock_sub, capsys, platform):
```

Comment:
I added the platform to all of the `test_call` cases just to ensure that `final_kwargs` doesn't do anything unexpected in the default case.